### PR TITLE
Release version 1.2.0

### PR DESCRIPTION
**Pull request by mhdostal**
Everything for the v1.2 release:

- LayerContents
- Bookmarks
- Toolkit integration
- update screen shots in readme
- Toolkit incorporation via git submodules

### CODE REVIEW

File: data-collection/data-collection/View Controllers/Layer Contents View Controllers/LayerContentsViewController.swift [hunk 1]
```diff
@@ -0,0 +1,498 @@
...
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+/// Describes a `LayerContentsViewController` configured to display a table of contents.
+/// - Since: 100.8.0
+public class TableOfContentsViewController: LayerContentsViewController {
+    override public init() {
+        super.init()
+        configuration = LayerContentsViewController.TableOfContents()
```
Comment (esreli) [inline data-collection/data-collection/View Controllers/Layer Contents View Controllers/LayerContentsViewController.swift:85]:
```suggestion
        configuration = TableOfContents()
```

File: data-collection/data-collection/View Controllers/Layer Contents View Controllers/LayerContentsViewController.swift [hunk 2]
```diff
@@ -0,0 +1,498 @@
...
+    /// - Since: 100.8.0
+    var title: String { get }
+}
+
+/// Describes a `LayerContentsViewController` configured to display a legend.
+/// - Since: 100.8.0
+public class LegendViewController: LayerContentsViewController {
+    override public init() {
+        super.init()
+        configuration = LayerContentsViewController.Legend()
```
Comment (esreli) [inline data-collection/data-collection/View Controllers/Layer Contents View Controllers/LayerContentsViewController.swift:72]:
```suggestion
        configuration = Legend()
```

**Comment by mhdostal** [comment]
> ### Toolkit Warnings
> There are 22 warnings associated with the Toolkit. Would it be possible to resolve these warnings? I'm afraid it's not good for our customers and not a good look for Data Collection.
> 
> I'm using toolkit commit `b57fc2ee2372a2f32aea7e9fa9948dc6eec8a763`

What version of Xcode/iOS are you building for?  I don't see any warnings on my end.

**Comment by esreli** [comment]
https://github.com/Esri/data-collection-ios/pull/232#issuecomment-632148288
@mhdostal 

Xcode 11.4.1
iOS 13.3 Simulated on iPhone 11

The 22 warnings:

20 warnings are identical to this one for different `TimeSlider` properties:

> /Users/eli9422/Documents/runtime/example/data-collection/ios/data-collection-ios/arcgis-runtime-toolkit-ios/Toolkit/ArcGISToolkit/TimeSlider.swift:2216:5: Lower ACL than parent Violation: Ensure definitions have a lower access control level than their enclosing parent (lower_acl_than_parent)

2 warnings are identical to this one for `JobManager`:

> /Users/eli9422/Documents/runtime/example/data-collection/ios/data-collection-ios/arcgis-runtime-toolkit-ios/Toolkit/ArcGISToolkit/JobManager.swift:202:12: Attributes Violation: Attributes should be on their own lines in functions and types, but on the same line as variables and imports. (attributes)

**Comment by mhdostal** [comment]
> Because Runtime introduced support for Metal at 100.8, I was unable to test the application using an iOS 12 simulator. Have you verified the app works as expected on a device running iOS 12?

Not on a device; all my devices are iOS 12.

**Comment by mhdostal** [comment]
@esreli - The Toolkit warnings you're encountering may be caused by an outdated SwiftLint.  Update and see if that helps.

**Comment by mhdostal** [comment]
> ### iOS 13 WorkMode contextualized navigation bar issue

I implemented your changes.

File: docs/index.md [hunk 3]
```diff
@@ -84,6 +85,26 @@ If a user elects to delete the offline map, the app deletes the offline mobile m
...
+
+Web maps can include a list of bookmarks. Each bookmark consists of a map extent (visible area) and a name. Bookmarks can be authored in ArcGIS Pro and the ArcGIS Web Map Viewer.
+
+You can click the Bookmark item in Extras to see a list of bookmarks in the map. Selecting a bookmark will show that bookmark's extent.
+
+![Screenshot showing bookmarks on iPhone](/docs/images/component-bookmarks-iphone.png)
+
+### View the map's layers
+
+You can click the Layers item in Extras to view the symbology for each layer.  You can "flip" the switch control to hide or show each layer.  Clicking the "chevron" button to the left of the layer name will hide or show the symbology, if any, for each layer.
```
[the remark this answers is not in the document]
Comment (mhdostal) [reply docs/index.md:104]: I think `select` is fine for the first change, but `Tapping` is better for the 2nd.

**Comment by esreli** [comment]
The testing report for iOS 12 found no iOS 12 specific issues.

**Pull request merged**